app/move/views/unassign.njk

```diff
@@ -0,0 +1,25 @@
+{% extends "form-wizard.njk" %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+    text: t("actions::back_to_allocation"),
+    href: "/allocation/" + allocation.id
+  }) }}
+
+  {{ super() }}
+{% endblock %}
+
+{% block contentMain %}
+  <p class="govuk-lede">
+    {{ t("allocation::unassign.detail", {
+      name: person.fullname,
+      move: move,
```

Comment:
This isn't being used by the looks of it, so can be dropped?

Reply:
It is being used.

Reply:
Sorry, I meant `move: move`. That value isn't being used in the translation so we can remove it from being passed into this `t()` call.

Reply:
👍